### ISSUE TITLE
Heading in OSC Proposed New Element draft proposal document for federal reporting file spec

### ISSUE THREAD

**Issue by johnmannpaul** [CEDStandards/CEDS-Elements#939]
So many new element proposals directly or indirectly reference a need based on federal reporting. When this is the case it would be nice to be able to easily find the a URL to the referenced file spec within the proposal itself or even just the official name of the report.  This information does often find its way into "Background Information" and "Other Standards Organizations Analysis including Legislation", but it would be helpful to have a heading (e.g. Related Federal Reporting File Specification) in the proposal itself that would explicitly reflect this information on a pending draft proposal.

**Comment by gilbertcj6** [comment]
Agreed.    Literally calling it out would make it more likely for users to remember to include it.  Are you thinking something like this: 

** Federal file specification name and/or link (if applicable) **

**Comment by johnmannpaul** [comment]
Yes, I think a heading like that in the proposal would be perfect.

**Comment by CJ-Gilbert-CEPI** [comment]
@jackie-hughes @AEMDuaneBrown 

This isn't really a change to the CEDS standard but a suggestion to tweak the process for proposing changes. 

Is this something that the CEDS team would review and acts upon?  Or does it have to go to the wider community in some way?

**Comment by jackie-hughes** [comment]
@johnmannpaul  @CJ-Gilbert-CEPI Thank you for your suggestions and engagement with the community. We agree that noting whether an issue relates to federal reporting, state-level reporting, Other within the issue itself would be beneficial for everyone. The CEDS team will work on updating the existing issue form and looks forward to sharing the revised version at the January OSC meeting.
